osgar/drivers/oak_camera.py

```diff
@@ -237,11 +290,11 @@ def run_input(self):
             g_logger.error("No stream enabled!")
             return
 
-        if self.cam_ip and cam_is_available(self.cam_ip):  # USB cameras?
-            device_info = dai.DeviceInfo()
-            device_info.state = dai.XLinkDeviceState.X_LINK_BOOTLOADER
-            device_info.desc.protocol = dai.XLinkProtocol.X_LINK_TCP_IP
-            device_info.desc.name = self.cam_ip
+        if self.cam_id and cam_is_available(self.cam_id):
+            device_info = dai.DeviceInfo(self.cam_id)
+            # device_info.state = dai.XLinkDeviceState.X_LINK_BOOTLOADER
```

Comment:
device_info is not used?

Reply:
This is not clear yet. I would like to test poe cameras today and then i will know

Reply:
It was updated according to this procedure: https://docs.luxonis.com/software/depthai-components/device/
It seems to be working.